dev/fattest.simplicity/src/componenttest/topology/utils/tck/TCKRunner.java

```diff
@@ -880,71 +911,45 @@ private static String jarPathInDir(String jarNameFragment, String dir) {
         return null;
     }
 
-    /**
-     * Run a command using a ProcessBuilder.
-     *
-     * @param  cmd
-     * @param  workingDirectory
-     * @param  outputFile
-     * @return                  The return code of the process. (TCKs return 0 if all tests pass and !=0 otherwise).
-     * @throws Exception
-     */
-    private static Process startProcess(String[] cmd, File workingDirectory) throws IOException, InterruptedException {
-        ProcessBuilder pb = new ProcessBuilder(cmd);
+    private static Map<String, String> getMvnEnv(File mavenUserHome) throws IOException, InterruptedException {
+        Map<String, String> mvnEnv = new HashMap<>();
 
-        // Enables timestamps in the mvnOutput logs
-        Map<String, String> mvnEnv = pb.environment();
         mvnEnv.put("MAVEN_OPTS", "-Dorg.slf4j.simpleLogger.showDateTime=true" +
                                  " -Dorg.slf4j.simpleLogger.dateTimeFormat=HH:mm:ss,SSS");
 
-        pb.directory(workingDirectory);
-        pb.redirectErrorStream(true);
-
-        Log.info(c, "runCmd", "Running command " + Arrays.asList(cmd));
-
-        Process process = pb.start();
-        return process;
-    }
-
-    private static void assertNotTimedOut(boolean timeout, ArrayList<String> output, long hardTimeout, long softTimeout) {
-        if (timeout) {
-            ArrayList<String> lastLines = new ArrayList<String>();
-            int numOfLinesToInclude = 7; // We will include the last 7 lines of the output file in the timeout message
-            int lineNum = output.size() - numOfLinesToInclude;
-            if (lineNum < 0) {
-                lineNum = 0;
-            }
-            while (lineNum < output.size()) {
-                lastLines.add(output.get(lineNum));
-                lineNum++;
+        if (mavenUserHome != null) {
+            String artifactoryServer = TCKUtilities.getArtifactoryServer();
```

Comment:
I think we looked at this before but should there be a `TCKUtilities.useArtifactory()` guard around these blocks?

Reply:
`mavenUserHome` will always be null if `useArtifactory()` returned false ... so the code is correct, just perhaps needs a comment